src/nspawn/nspawn-mount.c

```diff
@@ -435,6 +433,11 @@ int mount_all(const char *dest,
                                 o = options;
                 }
 
+                if (streq_ptr(mount_table[k].where, "/sys/fs/selinux") && is_dir("/sys/fs/selinux", true) < 0) {
+                        log_debug("SELINUX not active, skipping selinux mounts.");
+                        continue;
+                }
```

Comment:
this sounds unnecessary. the "fatal" bool in the table for the two selinux entries is "false", hence we try the mount, it fails, but we don't mind.

Or are you saying that the log messages generated in this case have a too high log level?